hack/terraform-eks.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: terraform enviroment variables need to be set
+AWS_CLI="aws"
+TERRAFORM="terraform"
+KUBECTL="kubectl"
+AWS_IAM="aws-iam-authenticator"
+RUN_PATH="$HOME/tmp/terraform-eks"
+CONFIG_PATH=$(dirname $(readlink -f "$0"))/terraform/eks
```

Comment:
it bothers me that we still don't have a portable solution that can work on macOS, since that's what a lot of devs in the K8s community use.

I am still confused as to why we need symbolic link support since even your `hack/install-cloud-tools.sh ` does not create any symbolic link. But assuming you still need that support, it seems that having the following would be nice:
```
if [[ "$OSTYPE" == "darwin"* ]]; then
   // check for greadlink
  command -v greadlink >/dev/null 2>&1 || (echo "Please install 'greadlink' with 'brew install coreutils'"; exit 1)
  CONFIG_PATH=$(dirname $(greadlink -f "$0"))/terraform/eks
else
  CONFIG_PATH=$(dirname $(readlink -f "$0"))/terraform/eks
fi
```

Reply:
The symlink is useful, when we have something like,
ln -s ~/go/src/.../antrea/hack/eks-terraform.sh /usr/local/bin/eks

then we can use the script anywhere. it is handy when u need to access eks cluster with little typing; i.e "eks kubectl ...."

I thought macOS is already supported, and I am assuming MAC users has some knowledge to install linux core utils tools. i.e
"brew install coreutils --with-default-names"

Reply:
I think the key point is to use check greadlink or readlink before using it on MAC or Linux.
I did not do exactly as suggested, but it should address the key point.